.github/workflows/doc.yml

```diff
@@ -0,0 +1,53 @@
+name: Doc 
+
+on:
+  push:
+    branches:    
+      - fix_path
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Install apt dependencies
+      run: sudo apt-get install -y libpq-dev
+    - name: Set up Python 3.9 
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+    - name: Set Variables
+      id: set_variables
+      run: |
+        echo "::set-output name=PY::$(python -c 'import hashlib, sys;print(hashlib.sha256(sys.version.encode()+sys.executable.encode()).hexdigest())')"
+        echo "::set-output name=PIP_CACHE::$(pip cache dir)"
+    - name: Cache PIP
+      uses: actions/cache@v2
+      with:
+        path: ${{ steps.set_variables.outputs.PIP_CACHE }}
+        key: ubuntu-pip-${{ steps.set_variables.outputs.PY }}
+    - name: Install dependencies
+      # pip install . --> for doc generate
+      run:  |
+        python -m pip install --upgrade pip
+        pip install -r requirements-dev.txt
+        pip install .
+    - name: Make doc
+      run: |
+         sphinx-build -M html doc/source build 
+         mkdir -p public
```

Comment:
line 25-39 can be simply changed to `tox -e docs` I think?

Reply:
done